src/prefect/core/flow.py

```diff
@@ -156,7 +155,7 @@ def __init__(
 
         self.name = name
         self.schedule = schedule
-        self.environment = environment or prefect.environments.CloudEnvironment()
+        self.environment = environment
```

Comment:
What if we made this `RemoteEnvironment` since it's minimal and actually functions?